routes/conversation.js

```diff
@@ -83,6 +82,26 @@ router.put('/read/:id', auth, async (req, res) => {
   }
 });
 
-router.delete('/delete/:id', (req, res) => {});
+// get all conversations. Given a search parameter (optional)
+router.get('/search/:searchterm', auth, async (req, res) => {
```

Comment:
we don't want to introduce a new route for this. instead reuse /users/conversation route

Reply:
https://github.com/hatchways/team-cocoa-puffs/blob/dev/routes/user.js#L172